webplugin/css/app/mck-sidebox-1.0.css

```
@@ -80,6 +80,22 @@
     box-shadow: 0 1.5rem 2rem rgba(0, 0, 0, 0.3);
 }
 
+/* Container mode: when widget is embedded in a custom container */
+.mck-sidebox.km-container-mode {
+    position: absolute !important;
+    top: 0 !important;
+    left: 0 !important;
+    right: 0 !important;
+    bottom: 0 !important;
```

Stylelint (17.2.0)

[error] Expected shorthand property "inset" (declaration-block-no-redundant-longhand-properties)

(declaration-block-no-redundant-longhand-properties)